extra/nginx

```diff
@@ -0,0 +1,72 @@
+#Using unix socket as upstream server
+upstream pa {
+	server unix:/tmp/planarally.sock fail_timeout=0;
+}
+
+#Forward http to https
+server {
+    listen 80;
+    listen [::]:80; #IPV6
+    server_name dnd.example.com;
+    return 301 http://www.$server_name$request_uri; 
+}
+
+
+server {
+	listen 443 ssl http2;
+	listen [::]:443 ssl http2; #IPV6
+
+	index index.html index.htm index.php index;
+  
+	client_max_body_size 4G;
+  server_name dnd.example.com;
+	
+	#protection against abuse - without it anyone can create their account
+	satisfy any;
+  #httpaswd for anyone outside local network
+	auth_basic	"Enter credentials";
+	auth_basic_user_file /path/to/.httpaswd;
+  #allow from local network - adjust for your network
+	allow 192.168.0.0/16; 
+	allow fe80::/10;
```

Comment:
I'm not familiar with this, but that's interesting!  It should become a server configuration at one point to disable new account creations etc.

I would maybe consider commenting this out as an example rather than as a default thing though